Competition2018/src/main/java/competition/subsystems/wrist/WristSubsystem.java

```diff
@@ -281,12 +281,14 @@ public void updatePeriodicData() {
         motor.updateTelemetryProperties();
         wristCalibratedProp.set(calibrated);
         currentWristAngleProp.set(getWristAngle());
-        lowerLimitProp.set(lowerLimitSwitch.get());
-        upperLimitProp.set(upperLimitSwitch.get());
-
+        
+        if (contract.isWristLimitsReady()) {
+          lowerLimitProp.set(lowerLimitSwitch.get());
+          upperLimitProp.set(upperLimitSwitch.get());
+        }/*
```

Comment:
In theory not needed... but the ZED still got hurt. Was it the wrist or something on the field?

Reply:
I think it was a falling cube :laughing: